userProfile/__tests__/functionality_unit.test.mjs

```diff
@@ -0,0 +1,57 @@
+import {findUserById} from '../userProfile.mjs'
+import {validateID} from '../../utilities/shared.mjs'
+
+// These test the pieces of functionality in the route that make it work.
+describe('userProfile endpoint functionality unit test (just testing helper functions). #functions_unit', () => {
+
+  it('No TPEN3 user ID provided.  User ID validation must be false.', () => {
+    expect(validateID()).toBe(false)
+  })
+  
+  it('Throws error and handles if id is 111', async () => {
+    try {
+      const res = await findUserById(111)
+      expect(true).toBe(false)
+    } catch (error) {
+      expect(error.message).toBe('Internal Server Error')
+    }
+  })
+  
+  it('Throws error and handles if id is 222', async () => {
+    try {
+      const res = await findUserById(222)
+      expect(true).toBe(false)
+    } catch (error) {
+      expect(error.message).toBe('Internal Server Error')
+    }
+  });
```

Comment:
remove superfluous `;` and any others like it.

Reply:
removed , Thanks for letting me know.